### PR TITLE
[IIIF-524] Update CSV parsing

### PR DESCRIPTION
**Pull request by ksclarke**
* Update CSV parsing and fix vague parsing error exception message
* Fix logging configuration for the live test runner
* Fix delete job handler's successful status response message
* Remove obsolete CDN property (missed from a previous commit)
* Add live test runner mount point example in the README
* Fix occurrence of duplicate IIIF Access URL columns [IIIF-515]
* Simplify setting of a subsequent run (part of JobFactory now)
* Add STRUCTURAL and MISSING as workflow status types
* Update and simplify file handling (and paths) in Item
* Move JobFactory into same package as Job (instead of in utils)
* Add object type + viewingHint coordination for STRUCTURAL status
* Lay the foundation for processing errors to be output to user                                                    
* Stop using canonical paths for performance improvement over SMB
* Improve fake-lambda.sh to make it more useful for subsequent runs

### CODE REVIEW

File: src/main/java/edu/ucla/library/bucketeer/handlers/LoadCsvHandler.java [hunk 1]
```diff
@@ -139,10 +141,10 @@ public void handle(final RoutingContext aContext) {
             }
 
             try {
-                final Job job = JobFactory.getInstance().createJob(jobName, new File(filePath));
+                final Job job = JobFactory.getInstance().createJob(jobName, new File(filePath), subsequentRun);
 
                 // Set the Slack handle and whether the job is a subsequent or initial run
```
Comment (hardyoyo) [inline src/main/java/edu/ucla/library/bucketeer/handlers/LoadCsvHandler.java]: this comment is no longer correct

Comment (ksclarke) [reply src/main/java/edu/ucla/library/bucketeer/handlers/LoadCsvHandler.java]: Pushed an update, thanks.

File: docs/loading-CSVs.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+* `missing` to `empty` (Rows without files in the first batch will be retried to see if the file now exists)
+* `ingested` to `ingested` (Something that is ingested is something that has been through two batch runs already)
+* `succeeded` to `ingested` (These are things that were converted in the first run and don't need to be tried again)
+
+Both initial and subsequent jobs will output a CSV file with the `Bucketeer State` and `IIIF Access URL` columns. Any row that is not there for structural purposes only should have a `failed`, `missing`, `ingested`, or `succeeded` state value.
+
+### What Are Structural Rows?
+
+Bucketeer's documentation talks about structural rows. Structural rows are rows in the CSV file that aren't intended to have a source file that needs to be processed. An example of this may be a row that represents a Collection. A CSV file may also represent a complex object, something that might have a Collection, a Work, and Pages. In this case, neither a Collection nor a Work might have source files. So, how does Bucketeer distinguish between a simple Work that's in a Collection and a Work that's in a collection but that also has pages?
```
Comment (hardyoyo) [inline docs/loading-CSVs.md]: consider striking the first sentence: "Bucketeer's documentation talks about structural rows."

Comment (ksclarke) [reply docs/loading-CSVs.md]: done

**Pull request merged**